tags.go

```diff
@@ -215,23 +215,29 @@ func serializeTags(name string, tags map[string]string) string {
 	const sep = "="
 
 	// discard pairs where the tag or value is an empty string
+	validTags := len(tags)
```

Comment:
Rename to numValid?

Reply:
Good call, fixed! Also, will need another approval since this is OSS and the code changed for the rename.